policyengine_us/parameters/gov/states/al/tax/income/deductions/itemized/excluded_deductions.yaml

```
@@ -0,0 +1,15 @@
+description: Itemized deductions excluded from Alabama.
+values:
+  2021-01-01:
+    - casualty_loss_deduction
+    - long_term_care_premium_deduction
+
+
+metadata:
+  unit: list
+  label: Itemized deductions excluded from Alabama.
+  reference:
+    - title: 2022 Schedule A (Form 1040)
+      href: https://www.revenue.alabama.gov/ultraviewer/viewer/basic_viewer/index.html?form=2023/01/22f40schabdc_blk.pdf
```

need to add paged to all pdf links. f.e. here:
```suggestion
      href: https://www.revenue.alabama.gov/ultraviewer/viewer/basic_viewer/index.html?form=2023/01/22f40schabdc_blk.pdf#page=1
```